### ISSUE TITLE
Email Archive policy advice is ambiguous

### ISSUE THREAD

**Issue by gbms311** [MicrosoftDocs/microsoft-365-docs#8183]
When researching the recommended approach to Email Archive policies, the references are circular and contradictory.

The below article says to use MRM, then it says MRM is old , don't use MRM - and points you to an article named, "Use retention policies and retention labels instead of older features" ... BUT, then four articles down it only uses MRM to create Archive policies. 

ie ...

_https://docs.microsoft.com/en-us/microsoft-365/compliance/archive-mailboxes?view=o365-worldwide_
**Learn about archive mailboxes**
- Users can access and store messages in their archive mailboxes by using Outlook and Outlook on the web. Users can also move or copy messages between their primary mailbox and their archive mailbox.

- Messages can also be moved to the archive mailbox by the [default Exchange retention policy](https://docs.microsoft.com/en-us/exchange/security-and-compliance/messaging-records-management/default-retention-policy) from Messaging Records Management (MRM).

- Note:  MRM, like Microsoft 365 retention policies and retention labels, can also automatically delete emails after a specified period. As an older technology than Microsoft 365 retention, MRM continues to work side-by-side with retention policies and retention labels from Microsoft 365 Compliance. For more information, see 'Use retention policies and retention labels instead of older features'

**Use retention policies and retention labels instead of older features**
_https://docs.microsoft.com/en-us/microsoft-365/compliance/retention?view=o365-worldwide#use-retention-policies-and-retention-labels-instead-of-older-features_
- If you need to proactively retain or delete content in Microsoft 365 for information governance, we recommend that you use retention policies and retention labels instead of the following older features.

Four articles down from the first menu link, the advice is "use MRM"

**Customize an archive and deletion policy for mailboxes in your organization**
_https://docs.microsoft.com/en-us/microsoft-365/compliance/set-up-an-archive-and-deletion-policy-for-mailboxes?view=o365-worldwide_

- Microsoft 365 compliance admins can create an archiving and deletion policy that automatically moves items to a user's archive mailbox and automatically deletes items from the mailbox.

- You do this by creating a Messaging Records Management (MRM) retention policy that's assigned to mailboxes, and moves items to a user's archive mailbox after a certain period of time and that also deletes items from the mailbox after they reach a certain age limit.

- The actual rules that determine what items are moved or deleted and when that happens are called retention tags. Retention tags are linked to an MRM retention policy, that in turn is assigned to a user's mailbox. .....

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: bb77538c-18e2-16c0-03cc-21b12c2b6b98
* Version Independent ID: bb77538c-18e2-16c0-03cc-21b12c2b6b98
* Content: [Learn about archive mailboxes for Microsoft 365 Compliance - Microsoft 365 Compliance](https://docs.microsoft.com/en-us/microsoft-365/compliance/archive-mailboxes?view=o365-worldwide)
* Content Source: [microsoft-365/compliance/archive-mailboxes.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/compliance/archive-mailboxes.md)
* Service: **o365-seccomp**
* GitHub Login: @cabailey
* Microsoft Alias: **cabailey**

**Comment by cabailey** [comment]
Hi @gbms311  - Although you got the right page & section, you missed out the information that explains both of these recommendations and why they aren't contradictory:

"... if you use the following MRM features, be aware that they aren't currently supported by Microsoft 365 retention policies:

- An archive policy for archive mailboxes to automatically move emails from a user's primary mailbox to their archive mailbox after a specified period of time. An archive policy (with any settings) can be used in conjunction with a Microsoft 365 retention policy that applies to a user's primary and archive mailbox.

- Retention policies applied by an admin to specific folders within a mailbox. A Microsoft 365 retention policy applies to all folders in the mailbox. However, an admin can configure different retention settings by using retention labels that a user can apply to folders in Outlook as a default retention label."

In other words, these two technologies don't (yet) have complete parity. If you want to move email from the current mailbox to the archive mailbox, you have to use MRM.  M365 retention can't do this (yet). What M365 retention can do, is retain email if needed for compliance, and/or automatically delete it after a specified time.  MRM can also do this. So overall, the recommendation is to use MRM only if you have to, because the feature isn't yet supported by M365 retention - and moving email to the archive mailbox is the most common reason.

**Comment by yogkumgit** [comment]
@gbms311 Hope this comment is helpful for you. If you see a documentation update is required, please feel free to open an issue for the same. We proceed here to close it. Thanks for taking out some time to open the issue. Appreciate and encourage you to do the same in future also.
